packages/blade/src/components/BottomSheet/BottomSheetCommon.tsx

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable react/display-name */
+import React from 'react';
+import type { ReactDOMAttributes } from '@use-gesture/react/dist/declarations/src/types';
+import { useBottomSheetContext } from './BottomSheetContext';
+import BaseBox from '~components/Box/BaseBox';
+import { IconButton } from '~components/Button/IconButton';
+import { CloseIcon } from '~components/Icons';
+import type { BladeElementRef } from '~src/hooks/useBladeInnerRef';
+import { isReactNative } from '~utils';
+
+type BottomSheetEmptyHeaderProps = Pick<
+  ReactDOMAttributes,
+  | 'onClickCapture'
+  | 'onKeyDown'
+  | 'onKeyUp'
+  | 'onLostPointerCapture'
+  | 'onPointerCancel'
+  | 'onPointerDown'
+  | 'onPointerMove'
+  | 'onPointerUp'
+>;
+
+const BottomSheetEmptyHeader = React.forwardRef<BladeElementRef, BottomSheetEmptyHeaderProps>(
+  (
+    {
+      onClickCapture,
+      onKeyDown,
+      onKeyUp,
+      onLostPointerCapture,
+      onPointerCancel,
+      onPointerDown,
+      onPointerMove,
+      onPointerUp,
+    },
+    ref,
+  ) => {
+    const { close } = useBottomSheetContext();
+    return (
+      <BaseBox
+        position="relative"
+        height="24px"
```

Comment:
No token for this 😸 ?

Reply:
Yeah so it's kind of hard coded even on figma side, let me add a comment & use size tokens.